app/models/person.rb

```diff
@@ -37,6 +41,10 @@ def primary_email_address
     email_addresses.detect(&:primary?)&.address
   end
 
+  def primary_email_address=(email)
```

Comment:
How does this handle duplicates or finding if somebody already is in our records?